src/index.html

```diff
@@ -22,6 +22,465 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="header__logo-link"
+      >
+        <img
+          src="./images/logo.png"
+          alt="MOYO"
+        />
+      </a>
+
+      <nav class="header__nav">
+        <ul class="header__menu">
+          <li class="header__menu-item">
+            <a
+              href="#"
+              class="header__menu-link header__menu-link--is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="header__menu-item">
+            <a
+              href="#"
+              class="header__menu-link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="header__menu-item">
+            <a
+              href="#"
+              class="header__menu-link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="header__menu-item">
+            <a
+              href="#"
+              class="header__menu-link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="header__menu-item">
+            <a
+              href="#"
+              class="header__menu-link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="header__menu-item">
+            <a
+              href="#"
+              class="header__menu-link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="header__menu-item">
+            <a
+              href="#"
+              class="header__menu-link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="header__menu-item">
+            <a
+              href="#"
+              class="header__menu-link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="container">
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <img
+            src="images/imac.jpeg"
+            alt="iMac on white background"
+            class="card__image"
+          />
+          <div class="card__title">
+            <h1 class="card__title">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </h1>
+          </div>
+
+          <div class="card__code">
+            <p>Product code: 195434</p>
+          </div>
+
+          <div class="card__review-container">
+            <div class="stars stars--4">
```

Comment:
It seems like the stars block is repeated multiple times with the same modifier 'stars--4', which indicates that the number of active stars is the same for all products. If the number of active stars is dynamic, consider implementing a way to update the class modifier based on the product's rating.